src/index.css

```diff
@@ -37,6 +37,7 @@ blockquote {
 .components-tab-panel__tab-content {
 	flex: 1;
 	overflow: auto;
+	scrollbar-width: thin;
```

Comment:
This change results in scrollbars overlaying the content, rather than pushing it to the left to make room for the scrollbar. Overlaying helps to avoid layout shift when moving between sites with chats that scroll, and others that don't.

Reply:
Nice, I checked the scrollbar and I think it looks much better!